packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js

```diff
@@ -123,6 +122,7 @@ export default ({ api, coreSagas }) => {
         coreSagas.kvStore.eth.fetchMetadataEth,
         askSecondPasswordEnhancer
       )
+      yield put(actions.middleware.webSocket.xlm.startStreams())
```

Comment:
Does the stream need any xlm data from metadata (the next call), such as the user's account?

Reply:
@plondon `startStreams` does not need any account info. It simply hits the `open` fn in `HorizonStreamingService` which defines `onMessage` and `onError`.

`_subscribeToAccount` is fired from the following metadata call which initiates the stream with the fetched accountId.